lib/pushNotifications.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:io';
+
+import 'package:firebase_messaging/firebase_messaging.dart';
+
+class PushNotificationsManager {
```

Comment:
Can you move this to lib/components/pushNotifications/pushNotifications.dart

Reply:
pushNotificationsManager.dart sounds a litttlllle better